src/secp256k1/src/field_10x26_impl.h

```diff
@@ -809,11 +809,11 @@ SECP256K1_INLINE static void secp256k1_fe_sqr_inner(uint32_t *r, const uint32_t
      *  Note that [x 0 0 0 0 0 0 0 0 0 0] = [x*R1 x*R0].
      */
 
-    d  = (uint64_t)(a[0]*2) * a[9]
-       + (uint64_t)(a[1]*2) * a[8]
-       + (uint64_t)(a[2]*2) * a[7]
-       + (uint64_t)(a[3]*2) * a[6]
-       + (uint64_t)(a[4]*2) * a[5];
+    d  = ( (uint64_t)a[0]*2 ) * a[9]
+       + ( (uint64_t)a[1]*2 ) * a[8]
+       + ( (uint64_t)a[2]*2 ) * a[7]
+       + ( (uint64_t)a[3]*2 ) * a[6]
+       + ( (uint64_t)a[4]*2 ) * a[5];
     /* VERIFY_BITS(d, 64); */
```

Comment:
I wonder if this will change the results of key generation and/or signing/verification

Reply:
We should add extra tests for this if it's even a potential risk.

Reply:
secp256 arithmetic changes has been moved to ticket: https://github.com/pastelnetwork/pastel/pull/24